application/__init__.py

```diff
@@ -38,9 +39,33 @@ def check_if_token_blacklist(decrypted_token):
         json_token_identifier = decrypted_token['jti']
         return json_token_identifier in BLACKLIST
 
+    '''Error handlers'''
```

Comment:
Nice work. Consider putting all error handlers in one separate file in challenge 3

Reply:
Got it!